test/clearingHouse/ClearingHouse.accounting.xyk.test.ts

```diff
@@ -129,11 +127,12 @@ describe("ClearingHouse accounting verification in xyk pool", () => {
         await collateral.mint(taker3.address, takerCollateral)
         await deposit(taker3, vault, 100, collateral)
 
-        // initiate both the real and mocked timestamps to enable hard-coded funding related numbers
-        await initiateBothTimestamps(clearingHouse)
-
         // increase insuranceFund capacity
         await collateral.mint(insuranceFund.address, parseUnits("1000000", 6))
+
+        // initiate both the real and mocked timestamps to enable hard-coded funding related numbers
+        // NOTE: Should be the last step in beforeEach
```

Comment:
Not quite understand why we move it to the bottom. What's the purpose of this?

Reply:
We have two block timestamps in test. One is mocked block timestamp, which is for founding payment testing purpose. Another one is evm block timestamp.

To prevent flaky on funding payment tests, it's better to sync mocked block timestamp and evm block timestamp before running each test cases. That's why it should be placed at the bottom of beforeEach. To prevent other devs change the order, add the NOTE comments here for reminder.